README.md

```diff
@@ -18,11 +18,28 @@ To recompile, publish, etc., just type the following in the root project:
 
 where command is one of compile, clean, test, publish, publish-local, etc.
 
-To create a full release, point publishTo and credentials to the appropriate
-values, then from the root project type "^release".
+To create a release, point publishTo and credentials to the appropriate
+values, then from the root project type "^release". Please do not use "^publish",
+as some additional preparation is necessary.
+
+The command "^release" will deploy artifacts to the "typesafe/dbuild" Bintray
```

Comment:
Maybe, here we should have the first paragraph be:

```
to make a dbuild release:

1. type "^release"  (please do not use "^ publish", as some additional preparation is necessary)
2. Check <bintray url> to ensure files are as expected (Optional)
3. type "root/bintrayRelease" which will make the release public
```

Then the other information you have (like publishing docs) is useful.